x/sequencer/keeper/rotation.go

```diff
@@ -95,3 +105,39 @@ func (k Keeper) OnProposerLastBlock(ctx sdk.Context, proposer types.Sequencer) e
 	)
 	return nil
 }
+
+// setSuccessorForRotatingRollapp will assign a successor to the rollapp.
+// It will prioritize non sentinel
+// called when a proposer has finished their notice period.
+func (k Keeper) setSuccessorForRotatingRollapp(ctx sdk.Context, rollapp string) error {
```

Comment:
can't this just be `chooseSuccessor`

Reply:
it's called only in rotation flow, so I thought it will be more clear